packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+            <div slot="default">
+                <nimble-checkbox></nimble-checkbox>
+                <nimble-accordion>
+                    <nimble-accordion-item>
+                         <div slot="heading">Nested Accordion One</div>
+                            <div slot="default">
+                                <span>Nested Accordion One Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                    <nimble-accordion-item>
+                         <div slot="heading">Nested Accordion Two</div>
+                            <div slot="default">
+                                <span>Nested Accordion Two Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                </nimble-accordion>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+            <div slot="default">
+                <nimble-button>Text Button</nimble-button>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+#### Nimble Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `error-visible` - boolean
```

Comment:
individual accordion items will be able to be configured in an error state?
Maybe I'm missing it but does the figma document have any examples of what an accordion looks like with multiple accordion items? Like it's not clear to me if the different items would share a border or not (presumably not) and what is the spacing between them.

Also curious how the nested accordions work for all the borders. Do borders only render on the inner most accordions?

Reply:
That's how I interpreted Brandon's spec, but he doesn't have any examples of multiple accordion items so it's hard to be certain. I'll ask Brandon about this and the border sharing, I agree it's unclear how spacing would work for different appearances. 

Here's an example from FAST's component explorer where I added nested accordions- It seems like the borders render on every accordion.
![fastNestedAccordionBorder](https://github.com/ni/nimble/assets/103057880/583bcab9-9d5f-4591-ae06-babc0fd0e526)

Reply:
For purposes of the spec are we still waiting on feedback here?

- is error states for individual accordion items or the full accordion?
- how do borders behave for nested accordions?
-

Reply:
I added answers to [these questions](https://github.com/ni/nimble/pull/1371#discussion_r1282213834) in the newest spec commit 👍. Error-visible will be set on individual accordion items, and all nested accordions will have the `ghost` appearance.